packages/mdx/readme.md

```diff
@@ -1083,7 +1083,7 @@ what unified does: please read through the [`unifiedjs/unified`][unified] readme
 (the part until you hit the API section is required reading).
 
 `@mdx-js/mdx` is a unified pipeline — wrapped so that most folks don’t need to
-know about unified: [`core.js#L65`][core].
+know about unified, but can easily integrate their remark and rehype plugins.
```

Comment:
```suggestion
know about unified.
```

I’d say just drop it instead of a new sentence. The next bullet points describe the unified processor!